front/app/modules/commercial/moderation/admin/containers/index.tsx

```diff
@@ -488,6 +593,45 @@ const Moderation = memo<Props & InjectedIntlProps>(({ className, intl }) => {
             </EmptyMessage>
           </Empty>
         )}
+
+        <Modal
```

Comment:
Could wrap the Modal in the test completely, and make the test check setting.allowed, so the modal component doesn't need to be loaded if the setting is not allowed